tick/eval_benchmarks_test.go

```diff
@@ -0,0 +1,293 @@
+package tick_test
```

Comment:
Can we name the file `stateful_expr_benchmarks_test.go` since it only applies the the stateful expression  evaluation?